docs/blog/posts/2024-11-22-screenshot-driven-development.md

```diff
@@ -6,18 +6,12 @@ authors:
   - rizel
 categories:
   - Web Development
+social:
+  cards_layout_options:
+    title: "Screenshot-Driven Development"  
+    description: "AI Agent uses screenshots to assist in styling."
+    background_image: "../images/screenshot-driven-development-blog/screenshot-driven-development.png" 
```

Comment:
will the relative path resolve to the absolute path when shared?

Reply:
ah good catch